frontend/src/component/pages/shared/CreateShortLinkSection.scss

```diff
@@ -12,3 +13,7 @@
   line-height: 20px;
   margin: 10px;
 }
+
+.preference-toggles > .preference-toggle:first-child {
```

Comment:
```suggestion
.preference-toggles {
```

Reply:
I think we are separating the two subsections.

Reply:
Yeah it's definitely not needed anymore. Done